system_test/helpers/aest_docker_api.erl

```diff
@@ -207,6 +208,11 @@ container_logs(ID) ->
             throw({docker_error, maps:get(message, decode_json(Response))})
     end.
 
+commit(ID, Tag) ->
```

Comment:
Probably a comment what commit is used for? Of course, I could read the manual of dockers.

Reply:
Commit is the standard command in Docker to create an image from a container file system